streams/next.json

```diff
@@ -1,192 +1,182 @@
 {
     "stream": "next",
     "metadata": {
-        "last-modified": "2021-03-03T19:18:02Z"
+        "last-modified": "2021-03-17T15:11:45Z"
     },
     "architectures": {
         "x86_64": {
             "artifacts": {
                 "aliyun": {
-                    "release": "33.20210302.1.0",
+                    "release": "33.20210315.1.0",
                     "formats": {
                         "qcow2.xz": {
                             "disk": {
-                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/33.20210302.1.0/x86_64/fedora-coreos-33.20210302.1.0-aliyun.x86_64.qcow2.xz",
-                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/33.20210302.1.0/x86_64/fedora-coreos-33.20210302.1.0-aliyun.x86_64.qcow2.xz.sig",
-                                "sha256": "efe83c1876665fdf807ef2bc0c8d5cf2ac833c95601027e2d5e6872ba13105b2",
-                                "uncompressed-sha256": "7f1aedc4f00a72b40ca5d911fe42c9015e1a32d32b7c8302fe7a1be6b058585c"
```

Comment:
We're losing the `uncompressed-sha256` fields. Out-of-date `fedora-coreos-stream-generator`?

Reply:
Looks like the generator was never revendored to add the field, but it somehow slipped into the last release anyway.  https://github.com/coreos/fedora-coreos-stream-generator/pull/22